jujugui/static/gui/src/app/extensions/app-cookies-extension.js

```diff
@@ -34,14 +34,60 @@ YUI.add('app-cookies-extension', function(Y) {
   Cookies.prototype = {
 
     /**
+      Get the value for a cookie.
+
+      @method _getCookie
+      @param cookie {String} The name of the cookie.
+      @return {String} The value of the cookie.
+    */
+    _getCookie: function(cookie) {
+      let value = null;
+      document.cookie.split(';').some(pair => {
+        const parts = pair.split('=');
+        if (parts[0].trim() === cookie) {
+          value = parts.length == 2 && parts[1].trim() || '';
+          return true;
+        }
+      });
+      return value;
+    },
+
+    /**
+      Set a cookie. This is here so that we can test that the right values were
+      passed.
+
+      @method _setCookie
```

Comment:
should we rename `_getCookie` to `_getDocumentCookie` in that case for consistency?